src/netappfiles-preview/azext_netappfiles_preview/aaz/latest/netappfiles/volume/_create.py

```diff
@@ -0,0 +1,1162 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "netappfiles volume create",
+    is_preview=True,
+)
+class Create(AAZCommand):
+    """Create or Update a volume
+
+    Create the specified volume within the capacity pool
+
+    :example: Create an ANF volume
+        az netappfiles volume create -g group --account-name aname --pool-name pname --volume-name vname -l location --service-level "Premium" --usage-threshold 107374182400 --creation-token "unique-token" --protocol-types NFSv3 --vnet myvnet --subnet-id "/subscriptions/mysubsid/resourceGroups/myrg/providers/Microsoft.Network/virtualNetworks/myvnet/subnets/default" --rules '[{"allowed_clients":"0.0.0.0/0","rule_index":"1","unix_read_only":"true","unix_read_write":"false","cifs":"false","nfsv3":"true","nfsv41":"false"}]'
```

Comment:
Recommend using [shorthand syntax](https://github.com/Azure/azure-cli/blob/dev/doc/shorthand_syntax.md#combine-with-full-value-and-partial-value) for the complex argument value in examples of aaz commands
```suggestion
        az netappfiles volume create -g group --account-name aname --pool-name pname --volume-name vname -l location --service-level "Premium" --usage-threshold 107374182400 --creation-token "unique-token" --protocol-types NFSv3 --vnet myvnet --subnet-id "/subscriptions/mysubsid/resourceGroups/myrg/providers/Microsoft.Network/virtualNetworks/myvnet/subnets/default" --rules "[{allowed_clients:'0.0.0.0/0',rule_index:1,unix_read_only:true,unix_read_write:false,cifs:false,nfsv3:true,nfsv41:false}]"
```